libs/output-mapping/tests/Writer/Helper/SliceHelperTest.php

```diff
@@ -267,10 +274,10 @@ public function testSliceSources(): void
         sort($slicedDirectories);
 
         self::assertSame($originalSource->getFile()->getPathname(), $slicedDirectories[0]);
-        self::assertSlicedData('"123","Test Name"', $slicedDirectories[0]);
+        self::assertSlicedData('"123","Test Name"' . PHP_EOL, $slicedDirectories[0]);
```

Comment:
Nebylo by jednodušší tam dát trim? Nebo to má nějakej skrytej význam?

Reply:
Jo trim mě napadl. Ale pak jsem si říkal že `assertSlicedData` je zavádějící, protože vnitřně by to dělalo trim a někdo by se na to v budoucnu nachytal.

Reply:
budiž

Reply:
Prisel jsem na to kdyz ten slicer joinuje male slicy do sebe. Bez konce radku jsem dostal nevalidni vystupni csv. Tak jsem to preventine pridal vsude.